### ISSUE TITLE
Temperature in Celsius and not milliCelsius

### ISSUE THREAD

**Issue by anasqirat** [melexis/mlx90632-library#37]
Throughout the library documentation, it is mentioned that the calculated temperature is in milliCelsius. From my dry run calculations and sensor datasheet, I believe the calculated temperature is in degree celsius.

**Comment by Letme** [comment]
True, that milliCelsius is taken from kernel driver which is integer implementation of this driver. Feel free to contribute a pull request
